ouroboros-consensus-test/test-consensus/Test/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -71,13 +71,15 @@ import           Test.Util.Orphans.IOLike ()
 import           Test.Util.TestBlock
 import           Test.Util.Tracer (recordingTracerTVar)
 
+{- HLINT ignore "Use list literal" -}
+
 {-------------------------------------------------------------------------------
   Top-level tests
 -------------------------------------------------------------------------------}
 
 tests :: TestTree
 tests = testGroup "ChainSyncClient"
-    [ testProperty "chainSync"                 $ prop_chainSync
+    [ testProperty "chainSync"                   prop_chainSync
```

Comment:
Please restore the `$`, we know it's redundant, but it's for consistency with the other lines below. 

You'll see that the main reason we have redundant `$`s throughout consensus is for consistency.

Reply:
I put the `$` back.  Thanks!